modules/registrars/openprovider/OpenProvider/Logger.php

```diff
@@ -15,16 +15,22 @@ class Logger extends AbstractLogger
      */
     public function log($level, $message, $context = []): void
     {
+        $replaceVars = isset($context['request']['password']) ? [
+            $context['request']['password'],
+            htmlentities($context['request']['password'])
+        ] : [];
+
+        if (isset($context['response']['data']['token'])) {
+            array_push($replaceVars,$context['response']['data']['token']);
```

Comment:
Hey! It is good that you use array_push, but in that case its bettor to use:
 `$replaceVars[] = $context['response']['data']['token']`

![image](https://user-images.githubusercontent.com/51438569/140876325-0e68ecec-e5c5-4ed6-8803-f9b08c9d1141.png)

![image](https://user-images.githubusercontent.com/51438569/140877042-4472fa72-dcde-495c-85f4-4d91647af93e.png)

Reply:
thanks i have updated